time    timestamp
int8    level_2G4
int8    level_5G